output/cloud/expv2/output.go

```diff
@@ -112,13 +161,21 @@ func (o *Output) StopWithTestError(testErr error) error {
 	default:
 	}
 
-	// Drain the SampleBuffer and force the aggregation for flushing
+	// Drain the SampleBuffer
+	o.collectSamples()
+
+	// Force the aggregation for flushing
```

Comment:
A bad rebase? This operation is already called two lines below

Reply:
Indeed. Fixed in [7c41939](https://github.com/grafana/k6/pull/3100/commits/7c4193982f295285ced4a2198a122eb74d299492).